src/org/zaproxy/zap/extension/pscanrulesAlpha/CacheableScanner.java

```diff
@@ -498,7 +499,7 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
 				lifetimeFound = true;
 				lifetime = 60 * 60 * 24 * 365;
 				//a liberal heuristic was assumed, for which no actual evidence exists
-				freshEvidence = Constant.messages.getString(MESSAGE_PREFIX_STORABLE_CACHEABLE + "evidence.liberallifetimeheuristic");
```

Comment:
https://github.com/zaproxy/zap-extensions/pull/413/files#diff-dad61ab60235d119bf9d2e384ec8c568R365n – this line initializes it to null. I don't think re-initialization to null is required?